# MyLib

## workflow managers

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| Nextflow| cell| https://www.nextflow.io/, https://github.com/pmb59/bioinformatics-workflows/tree/master/nextflow, https://www.nextflow.io/docs/latest/index.html, https://nf-co.re/ |
| Galaxy| cell| https://github.com/pmb59/bioinformatics-workflows/tree/master/galaxy , https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html, https://training.galaxyproject.org/training-material/, https://docs.galaxyproject.org/en/master/|
| Snakemake| cell|https://github.com/pmb59/bioinformatics-workflows/tree/master/snakemake, https://snakemake.readthedocs.io/en/stable/tutorial/tutorial.html, https://snakemake.readthedocs.io/en/stable/ |
| SciPipe| cell| https://github.com/pmb59/bioinformatics-workflows/tree/master/scipipe, https://scipipe.org/writing_workflows/,https://scipipe.org/ |
| GenPipes| cell| https://github.com/pmb59/bioinformatics-workflows/tree/master/genpipes, https://genpipes.readthedocs.io/en/master/tutorials/list_tutorials.html, https://genpipes.readthedocs.io/en/master/index.html|
| Bpipe | cell| https://github.com/pmb59/bioinformatics-workflows/tree/master/bpipe|


## Informatic Tools/Workflows/languages

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| Nextflow: Data-driven computational pipelines | cell| https://www.nextflow.io/|
| Python | cell| https://www.python.org/|
| R | cell| https://www.r-project.org/|
| Bioconductor | cell| https://www.bioconductor.org/ |
| PyTorch: Deep learning library for python| cell| https://pytorch.org/|
| D3: The JavaScript library for bespokedata visualization| cell| https://d3js.org/|
| | cell| |
| | cell| |
| | cell| |


## models/software/pipelines

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| DeepFRI: Protein function prediction| cell| https://github.com/flatironinstitute/DeepFRI|
| MSA Transformer: Generate synthetic proteins statistically similar| https://doi.org/10.7554/eLife.79854| cell |
| GenProSeq: Generating Protein Sequences with Deep Generative Models| cell | GenProSeq: Generating Protein Sequences with Deep Generative Models|
| ESM models (META/facebook): Evolutionary Scale Modeling.| cell| cell |
| EvoMIL: Prediction of virus-host association. Prediction of virus-host association using protein language models and multiple instance learning | EvoMIL: Prediction of virus-host association. Prediction of virus-host association using protein language models and multiple instance learning | EvoMIL: Prediction of virus-host association. Prediction of virus-host association using protein language models and multiple instance learning |
| ExamPle: Explainable deep learning framework for the prediction of plant small secreted peptides. | cell| ExamPle: Explainable deep learning framework for the prediction of plant small secreted peptides. |
| SuMD: Supervised Molecular Dynamics Simulations. | cell|SuMD: Supervised Molecular Dynamics Simulations.  |
| S4PRED: A tool for accurate prediction of a protein's secondary structure from only its amino acid sequence with no evolutionary information i.e. MSA required| cell| https://github.com/psipred/s4pred|
| LinearDesign: Algorithm for Optimized mRNA Design Improves Stability and Immunogenicity| cell| https://github.com/LinearDesignSoftware/LinearDesign|
| gINTomics is an R package for Multi-Omics data integration and visualization| cell| https://github.com/angelovelle96/gINTomics|

## Useful web-tools

| Title  | field | Link |
| ------------- | ------------- | ----------- |
|VOSviewer: software tool for constructing and visualizing bibliometric networks | Content Cell  | VOSviewer: software tool for constructing and visualizing bibliometric networks |
|  Consensus: Evidence-Based Answers, Faster| Content Cell  |  |     Consensus: Evidence-Based Answers, Faster       |
| Connected papers: Explore connected papers in a visual graph| cell| Connected papers: Explore connected papers in a visual graph|
| | cell| |
| | cell| |
| | cell| |

## Books

| Title  | field | Link |
| ------------- | ------------- | ----------- |
|Data Analysis in Medicine and Health using R  | Content Cell  | https://bookdown.org/drki_musa/dataanalysis/             |
|  | Content Cell  |              |

## Blogs

| Title  | field | Link |
| ------------- | ------------- | ------------|
| codon | cell | https://www.readcodon.com/|
| Liams' Blog| cell |https://liambai.com/  |
| | cell | |
| | cell | |
| | cell | |
| | cell | |

## Blog articles

| Title  | field | Link |
| ------------- | ------------- | ------------|
| AlphaFold 2 is here: what’s behind the structure prediction miracle  | cell |https://www.blopig.com/blog/2021/07/alphafold-2-is-here-whats-behind-the-structure-prediction-miracle/  |
| The big problems | cell  | https://www.science.org/content/blog-post/big-problems |
|  Why we didn’t get a malaria vaccine sooner| Content Cell  |        https://worksinprogress.co/issue/why-we-didnt-get-a-malaria-vaccine-sooner      |
| How to represent a protein sequence | Content Cell  |      https://liambai.com/protein-representation/        |
| What we can learn from evolving proteins | Content Cell  |       https://liambai.com/protein-evolution/       |
|  | Content Cell  |              |
|  | Content Cell  |              |
|  | Content Cell  |              |

## papers

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| Generative models for protein structure: A comparison between Generative Adversarial and Autoregressive networks.| cell | https://webthesis.biblio.polito.it/15944/|
| | cell | |
| | cell | |
| | cell | |

## Tutorials/guides

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| Introduction to RNA-Seq using high-performance computing | Content Cell  | https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/)https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/  |
| Introduction to Single-Cell Analysis with Bioconductor | Content Cell  |  https://bioconductor.org/books/release/OSCA/ |
|Orchestrating Single-Cell Analysis with Bioconductor Authors: Robert Amezquita [aut], Aaron Lun [aut], Stephanie Hicks [aut], Raphael Gottardo [aut], Alan| cell | Orchestrating Single-Cell Analysis with Bioconductor Authors: Robert Amezquita [aut], Aaron Lun [aut], Stephanie Hicks [aut], Raphael Gottardo [aut], Alan|
| Development of an Online Laboratory Handbook and Remote Workflow for Chemistry and Pharmacy Master’s Students to Undertake Computer-Aided Drug Design.| cell | Development of an Online Laboratory Handbook and Remote Workflow for Chemistry and Pharmacy Master’s Students to Undertake Computer-Aided Drug Design. |
| Contact maps| cell | Contact maps|
| TeachOpenCADD: Virtual screening.| cell | TeachOpenCADD: Virtual screening.|
| GROMACS tutorial| cell | http://www.mdtutorials.com/gmx/|
|ESM models (META/facebook): Evolutionary Scale Modeling. | cell |ESM models (META/facebook): Evolutionary Scale Modeling. |
| The Art of Command Line| cell | https://github.com/crazyhottommy/the-art-of-command-line|
| RNA-seq analysis| cell | https://github.com/crazyhottommy/RNA-seq-analysis|
| | cell | |
| | cell | |

## wet lab

| Title  | field | Link |
| ------------- | ------------- | ----------- |
| basic methods | cell | https://www.youtube.com/@csberg5856/videos|
| | cell | |
| | cell | |
| | cell | |

